File: WebsiteAppRPG/WebsiteAppRPG.Core/Entities/Map.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebsiteAppRPG.WebsiteAppRPG.Core.Entities
{
    [Table("Maps")]
    public class Map
    {
        [Key]
        [Column("MapID")]
        public int MapId { get; set; }

        [Column("Name")]
        public string Name { get; set; }

        [Column("Width")]
        public int Width { get; set; }

        [Column("Height")]
        public int Height { get; set; }

        [Column("EntranceX")]
        public int EntranceX { get; set; }

        [Column("EntranceY")]
        public int EntranceY { get; set; }

        [Column("ExitX")]
        public int ExitX { get; set; }

        [Column("ExitY")]
        public int ExitY { get; set; }

        public Map(string name, int width, int height, int entranceX, int entranceY, int exitX, int exitY)
        {
            Name = name;

            Width = width;
            Height = height;

            EntranceX = entranceX;
            EntranceY = entranceY;

            ExitX = exitX;
            ExitY = exitY;
        }
    }
}

File: WebsiteAppRPG/WebsiteAppRPG.Frontend/website-app-rpg-frontend/src/models/map.ts
export class Map {

    public name: string;
    
    public width: number;
    public height: number;

    public entranceX: number;
    public entranceY: number;

    public exitX: number;
    public exitY: number;
    
    constructor(name: string, mapWidth: number, mapHeight: number, entranceX: number, entranceY: number, exitX: number, exitY: number) {
        this.name = name;

        this.width = mapWidth;
        this.height = mapHeight;

        this.entranceX = entranceX;
        this.entranceY = entranceY;

        this.exitX = exitX;
        this.exitY = exitY;
    }
}